src/Model/GroundWaterFlow/gwf3dis8.f90

```diff
@@ -1308,15 +1309,59 @@ subroutine connection_vector(this, noden, nodem, nozee, satn, satm, ihc, &
     return
   end subroutine
 
-  ! return discretization type
+  !> @brief return discretization type
   subroutine get_dis_type(this, dis_type)
     class(GwfDisType), intent(in) :: this
     character(len=*), intent(out) :: dis_type
 
     dis_type = "DIS"
-
   end subroutine get_dis_type
 
+  !> @brief Get a 2D array of polygon vertices
+  subroutine get_polyverts(this, ic, polyverts, wrap)
+    ! -- modules
+    use InputOutputModule, only: get_ijk
+    ! -- dummy
+    class(GwfDisType), intent(inout) :: this
+    integer(I4B), intent(in) :: ic !< cell number (reduced)
+    real(DP), allocatable, intent(out) :: polyverts(:, :) !< polygon vertices
+    logical(LGP), intent(in), optional :: wrap !< whether to wrap around (duplicating a vertex)
+    ! -- local
+    integer(I4B) :: icu, nverts, irow, jcol, klay
+    real(DP) :: cellx, celly, dxhalf, dyhalf
+    logical(LGP) :: lwrap
+
+    ! check wraparound option
+    if (.not. (present(wrap))) then
+      lwrap = .false.
+      nverts = 4
+    else
+      lwrap = wrap
+      if (lwrap) &
+        nverts = 5
+    end if
+
+    ! allocate vertices array
+    allocate (polyverts(nverts, 2))
```

Comment:
Should this be allocated and accessed as polyverts(2, nverts) to be consistent with Fortran column-major ordering?  Probably doesn't matter a whole lot, but it would make sense to be consistent with the rest of the code.

Reply:
switched to column-major indexing for `get_polyverts` as well as for `point_in_polygon` and its tests